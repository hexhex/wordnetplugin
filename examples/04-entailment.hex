%% this returns all entailments of a word
%% POS: verbs

%% the search number for entailment is 4

word("stumble").
entailment(W,E) :- &wordnet[4,W](_,_,E,_), word(W).

