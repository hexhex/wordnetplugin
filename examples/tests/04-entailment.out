{entailment("stumble","walk"), word("stumble")}
